contributing-guides/style-guide.ko.md

```diff
@@ -0,0 +1,165 @@
+# 스타일 가이드
+
+이 페이지는 `tldr` 페이지에 대한 형식 지정 지침들을 나열합니다.
+
+## 레이아웃
+
+각 페이지의 기본 포맷은 다음 템플릿과 일치해야 하며,<br>
+다음과 최대 8개의 명령어 예제를 포함해야 합니다:
+```md
+# 명령어 이름
+
+> 짧고 간결한 설명
+> 보통 1줄, 필요한 2줄 까지 허용됨
+> <https://example.com/command_name/help/page>에서 더 많은 정보를 찾을 수 있습니다. 
+
+- 코드 설명:
+
+`command_name options`
+
+- 코드 설명:
+
+`command_name options`
+
+...
+```
+
+예시:
+```md
+# 명령어 이름
+```
+# krita
+
+> krita는 디지털 아티스트를 위해 설계된 스케치/페인팅 프로그램입니다.
+> `gimp` 페이지도 참조하세요.
+> 더 많은 정보는 <https://docs.krita.org/en/reference_manual/linux_command_line.html>를 참조하세요.
+
+- krita 시작:
+
+`krita`
+
+- 로딩 화면 없이 시작:
+
+`krita --nosplash`
+
+- 특정 파일 열기:
+
+`krita {{path/to/image1 path/to/image2 ...}}`
+
+- 특정 workspace(`Animation`)에서 시작:
```

Comment:
Conventionally, we put spaces between full-width characters and half-width punctuations.
```suggestion
- 특정 workspace (`Animation`) 에서 시작:
```